props/blaster.h

```diff
@@ -301,22 +307,44 @@ class Blaster : public PropBase {
       case EFFECT_CLIP_IN: hybrid_font.PlayCommon(&SFX_clipin); return;
       case EFFECT_CLIP_OUT: hybrid_font.PlayCommon(&SFX_clipout); return;
       case EFFECT_RELOAD: hybrid_font.PlayCommon(&SFX_reload); return;
-      case EFFECT_MODE:
-	if (SFX_mode) {
-	  hybrid_font.PlayCommon(&SFX_mode);
-	  return;
-	}
-	// TODO: would rather do a Talkie to speak the mode we're in after mode sound
-	beeper.Beep(0.05, 2000.0);
-	return;
+      case EFFECT_MODE: SayMode(); return;
       case EFFECT_RANGE: hybrid_font.PlayCommon(&SFX_range); return;
       case EFFECT_EMPTY: hybrid_font.PlayCommon(&SFX_empty); return;
       case EFFECT_FULL: hybrid_font.PlayCommon(&SFX_full); return;
       case EFFECT_JAM: hybrid_font.PlayCommon(&SFX_jam); return;
       case EFFECT_UNJAM: hybrid_font.PlayCommon(&SFX_unjam); return;
       case EFFECT_PLI_ON: hybrid_font.PlayCommon(&SFX_plion); return;
       case EFFECT_PLI_OFF: hybrid_font.PlayCommon(&SFX_plioff); return;
-	
+  
+    }
+  }
+
+  void SayMode() {
+    switch(blaster_mode) {
```

Comment:
space after switch

Reply:
This whole section fixed and tested good.